### ISSUE TITLE
Render HTML within MD with HTML-allowing plugins?

### ISSUE THREAD

**Issue by fthommen** [clockoon/dokuwiki-plugin-commonmark#26]
You write "Since DW do not parses raw HTML without htmlok config, [HTML blocks](https://spec.commonmark.org/0.30/#html-blocks) is passed." (https://github.com/clockoon/dokuwiki-plugin-commonmark). However even with plugins which allow HTML ([htmlok](https://www.dokuwiki.org/plugin:htmlok) or [Config:HTMLOk](https://www.dokuwiki.org/plugin:confightmlok)), HTML is not parsed. We use extensively HTML e.g. to create structured lists within Markdown tables (which is compatible with GitLab flavoured MD) and we hoped to be able to integrate these pages in DW.

Such a block could look like
```
| X | Y |
| ----------- | ----------------- |
| <ul><li>A<li>B<ul><li>Ba<li>Bb</ul><li>C</ul> | me |
```

and render in Gitlab/MD like

| X | Y |
| ----------- | ----------------- |
| <ul><li>A<li>B<ul><li>Ba<li>Bb</ul><li>C</ul> | me |

but looks like this in DW with the Config:HTMLOk plugin and the Commonmark plugin:
<img width="352" alt="image" src="https://github.com/clockoon/dokuwiki-plugin-commonmark/assets/279232/87cacb7f-66bf-4a12-8991-28983b056219">

Are there plans to render (raw) HTML in the future?

**Comment by clockoon** [comment]
It is my mistake to remove HTML tags during parse, not just pass through. I will fix it shortly.

**Comment by clockoon** [comment]
The issue is fixed in 8f24791, and README have been updated for reflecting DW's HTML policy change in 93775a6.
